packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -55,12 +56,19 @@ class InstantSearch extends Component {
     const initialState = this.isControlled ? props.searchState : {};
     this.isUnmounting = false;
 
+    const stalledSearchDelay =
+      props.stalledSearchDelay === undefined ||
+      props.stalledSearchDelay === null
+        ? 200
+        : props.stalledSearchDelay;
```

Comment:
We could provide a default props for `stalledSearchDelay ` with the value `200`. It will avoid the need to do the condition.